plugin/ocgrpc/server_metrics.go

```diff
@@ -73,6 +74,14 @@ var (
 		Aggregation: view.Count(),
 	}
 
+	ServerStartedRPCsView = &view.View{
+		Measure: ServerStartedRPCs,
+		Name: "grpc.io/server/started_rpcs",
+		Description: "Count of started RPCs by method.",
```

Comment:
```suggestion
		Description: "Number of started server RPCs",
```
Consistent with https://github.com/census-instrumentation/opencensus-java/blob/2e90f499881f601bfee1942b8bbfa2f79eaccc34/contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViewConstants.java#L898

Reply:
Done. Kept period.